### DISCUSSION TITLE
Rust in Lambda@edge and CloudFront functions

### DISCUSSION THREAD

**Discussion by rimutaka** [aws/aws-lambda-rust-runtime#327] [Ideas]
This is just a request to AWS to allow us run binaries for Lambda@edge and CloudFront functions.
As of now, they only allow NodeJS and Python.

The billing there is per ms of execution. What can be better suited for that than Rust?

**Comment by snibbor** [comment]
Do you know if there has been any development planned for this idea? I am considering whether it is it worth trying to run a rust program from Python/NodeJS... any ideas to try for now? Thanks.
